From 336e69d6882a1b8d725c43f1c2e340ef4464bac5 Mon Sep 17 00:00:00 2001
From: Julien Lepiller <julien@lepiller.eu>
Date: Sat, 31 Dec 2016 11:21:14 +0100
Subject: [PATCH] fix OCAMLPATH in makefile

---
 Makefile            | 2 +-
 make/Makefile.ocaml | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 4db87da..9315887 100644
--- a/Makefile
+++ b/Makefile
@@ -23,7 +23,7 @@ OCAMLFIND_DESTDIR := $(shell cygpath -w $(OCAMLFIND_DESTDIR))
 endif
 export OCAMLFIND_DESTDIR

-OCAMLPATH := $(PIQI_ROOT)/deps
+OCAMLPATH := $(OCAMLPATH):$(PIQI_ROOT)/deps
 ifeq ($(SYSTEM),$(filter $(SYSTEM),mingw mingw64))
 OCAMLPATH := $(shell cygpath -w $(OCAMLPATH))
 endif
diff --git a/make/Makefile.ocaml b/make/Makefile.ocaml
index 2b6324d..b82c041 100644
--- a/make/Makefile.ocaml
+++ b/make/Makefile.ocaml
@@ -1,7 +1,7 @@
 OCAMLMAKEFILE := $(PIQI_ROOT)/make/OCamlMakefile


-OCAMLPATH := $(PIQI_BUILD)/lib/ocaml
+OCAMLPATH := $(OCAMLPATH):$(PIQI_BUILD)/lib/ocaml
 ifeq ($(SYSTEM),$(filter $(SYSTEM),mingw mingw64))
 OCAMLPATH := $(shell cygpath -w $(OCAMLPATH))
 endif
@@ -23,6 +23,7 @@ post_target::


 .ml.mli: $(EXTRADEPS)
+			echo $(OCAMLPATH)
 			$(QUIET)pp=`sed -n -e '/^#/d' -e 's/(\*pp \([^*]*\) \*)/\1/p;q' $<`; \
 			if [ -z "$$pp" ]; then \
 			  $(ECHO) $(REAL_OCAMLFIND) $(INTF_OCAMLC) $(OCAML_FIND_PACKAGES) \
--
2.11.0

